content/200-orm/200-prisma-client/600-observability-and-logging/240-metrics.mdx

```diff
@@ -64,7 +64,7 @@ npm install prisma@latest --save-dev
 npm install @prisma/client@latest --save
 ```
 
-### Step 2: Enable the feature flag in the Prisma schema file
+### 2. Enable the feature flag in the Prisma schema file
```

Comment:
I like the numbering instead of the `step 1`/`step 2`-- it's more consistent with the rest of our docs.